app/controllers/sessions_controller.rb

```diff
@@ -15,9 +15,8 @@ def sign_in_and_redirect(resource_or_scope, resource = nil)
     scope = Devise::Mapping.find_scope!(resource_or_scope)
     resource ||= resource_or_scope
     sign_in(scope, resource) unless warden.user(scope) == resource
-    sign_out_link = view_context
-      .link_to 'Sign out', destroy_user_session_path, method: :delete
-
+    sign_out_link = view_context.link_to(
+      'Sign out', destroy_user_session_path, method: :delete)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.